controller/rest/system.go

```diff
@@ -2607,23 +2621,43 @@ func _importHandler(w http.ResponseWriter, r *http.Request, tid, importType, tem
 			importTask.TotalLines = lines
 			importTask.Percentage = 3
 			importTask.LastUpdateTime = time.Now().UTC()
-			clusHelper.PutImportTask(&importTask)
+			_ = clusHelper.PutImportTask(&importTask) // Ignore error because progress update is non-critical
 			kv.SetImporting(1)
 			eps := cacher.GetAllControllerRPCEndpoints(access.NewReaderAccessControl())
 			switch importType {
 			case share.IMPORT_TYPE_CONFIG:
 				value := r.Header.Get("X-As-Standalone")
 				ignoreFed, _ := strconv.ParseBool(value)
-				go cfgHelper.Import(eps, localDev.Ctrler.ID, localDev.Ctrler.ClusterIP, login.domainRoles, importTask,
-					tempToken, revertFedRoles, postImportOp, rpc.PauseResumeStoreWatcher, ignoreFed)
+				go func() {
+					if err := cfgHelper.Import(eps, localDev.Ctrler.ID, localDev.Ctrler.ClusterIP, login.domainRoles, importTask,
+						tempToken, revertFedRoles, postImportOp, rpc.PauseResumeStoreWatcher, ignoreFed); err != nil {
+						log.WithFields(log.Fields{"error": err}).Error("Import")
+					}
+				}()
```

Comment:
Since the import task is a relatively long operation, the design stores its status and result (including any errors) in Consul. 
The caller (UI, REST client) will poll for the result. The outcome of the goroutine here is not critical.